### PR TITLE
adding new accessible styling to blog article pages

### PR DESCRIPTION
**Pull request by gregdorward**
As a follow up to the work @dstodolny did for the listings page, and to save his sanity in making more accessibility requests, I've applied the @badgerRedOnWhite colour to a number of elements on the blog article pages.

### CODE REVIEW

File: styles/_blog-article.less [hunk 1]
```diff
@@ -167,6 +167,7 @@
     img.arrow {
       width: 28px;
       height: 16px;
+      color: @badgerRedOnWhite;
```
Comment (chris-brookes) [inline styles/_blog-article.less]: You have added a colour style to an image. We need to change the image to get a new colour for the arrow.

**Comment by gregdorward** [comment]
@chris-brookes, I've removed the colouring of that image now if you could give it another check please?

**Comment by chris-brookes** [comment]
@gregdorward - We still have the old badger red on the text link hover. (line 71)

**Comment by gregdorward** [comment]
Cheers @chris-brookes, that should be sorted now

**Comment by chris-brookes** [comment]
@gregdorward - Now you're starting to see what it's like for the devs when testers look at your work.
"If only you'd tested your work properly, we wouldn't have to do this!" :-)

**Comment by gregdorward** [comment]
I think this can be merged when you have time please @dstodolny

**Pull request merged**